src/datatypes/user/NewUserForm.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Define type for new uer form
+ *
+ * @author Hyecheol (Jerry) Jang <hyecheol123@gmail.com>
+ */
+
+/**
+ * Interface for NewUserForm
+ */
+export default interface NewUserForm {
+  username: string;
+  password: string;
+  admissionYear: number;
+  nameKorean: string;
+  nameEnglish?: string;
```

Comment:
optional maybe?

Reply:
https://www.typescriptlang.org/docs/handbook/2/objects.html

Reply:
Runtime checkers are implemented separately

Reply:
That's why it is optional.